CondTools/CTPPS/plugins/WriteCTPPSTotemDAQMappingMask.cc

```diff
@@ -43,16 +43,13 @@ class WriteCTPPSTotemDAQMappingMask : public edm::one::EDAnalyzer<> {
   void analyze(const edm::Event &e, const edm::EventSetup &es) override;
 };
 
-using namespace std;
-using namespace edm;
```

Comment:
why is this preferred? I don't see mentions of this in the [code rules](http://cms-sw.github.io/cms_coding_rules.html).
I think the general directive in the core cpp guidelines about avoid putting `using` directives applies only to header files (`*.h`) because any file that includes that header will bring everything in the namespace into scope, which can cause name hiding and name collision problems that are very difficult to debug. 
But this module is a `plugin` (that by construction cannot be included anywhere). Can you clarify?

Reply:
Some cleanup, given that the two namespaces can be simply avoided as a whole by adding a `std::` in front of couple of strings?

Reply:
I really don't think this qualifies as a clean-up.